lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -492,6 +492,17 @@ function _dosomething_reportback_get_permalink_copy_vars($user) {
  * Alters the signup form that is attached to permalink pages.
  */
 function dosomething_reportback_form_alter(&$form, &$form_state, $form_id) {
+
+  // Make sure a default run is added to the filter
+  if ($form['#id'] == 'views-exposed-form-reportback-files-page-1') {
```

Comment:
if you dpm the form name and change the page you'll see it never changes. i know its weird but it seems fine